integration/stability/soak_parallel_rm.sh

```diff
@@ -48,6 +51,20 @@ check_vsock_active() {
 	fi
 }
 
+enable_netmon() {
+	echo "Enabling netmon at the configuration file"
+	sed -i 's|#enable_netmon = true|enable_netmon = true|' ${DEFAULT_KATA_CONFIG}
+	echo "Reloading unit files"
+	sudo systemctl daemon-reload
```

Comment:
Similar to the docker comment, you don't need to call this (here or below) as no systemd units are being touched.

Reply:
Historical reference: I think looong looong ago the addition of the systemctl reload and the docker restart were added at some generic outer level to try and ensure we had a 'known stable starting point' when running the metrics (as we really like repeatable results there).
As per other comments, we probably don't need to do that here.